scripts/generate_schemas.py

```diff
@@ -83,24 +84,25 @@ def formatSchema(self):
         stack = [schema]
         # Strip out all the docs
         while len(stack) > 0:
-            d = stack.pop()
-            if "doc" in d:
-                d["doc"] = ""
-            for u in d.values():
-                if isinstance(u, dict):
-                    stack.append(u)
-                elif isinstance(u, list):
-                    for v in u:
-                        if isinstance(v, dict):
-                            stack.append(v)
-        s = json.dumps(schema)
-        with tempfile.TemporaryFile() as f:
-            f.write(s)
-            f.seek(0)
+            elm = stack.pop()
+            if "doc" in elm:
+                elm["doc"] = ""
+            for value in elm.values():
+                if isinstance(value, dict):
+                    stack.append(value)
+                elif isinstance(value, list):
+                    for dic in value:
+                        if isinstance(dic, dict):
+                            stack.append(dic)
+        jsonData = json.dumps(schema)
+        with tempfile.TemporaryFile() as tmp:
+            tmp.write(jsonData)
+            tmp.seek(0)
             # Filter the text through fmt to make it look like acceptable code.
-            p = subprocess.Popen(["fmt"], stdout=subprocess.PIPE, stdin=f)
-        (output, err) = p.communicate()
-        exitStatus = p.wait()
+            subproc = subprocess.Popen(
+                ["fmt"], stdout=subprocess.PIPE, stdin=tmp)
+        (output, err) = subproc.communicate()
```

Comment:
I think these lines should be indented under the `with` clause, as we're could be using stdin at any point. (My bad in the first place)